community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -120,11 +120,7 @@ def per_instance_properties(node):
 
 def create_instances_request(nodes, partition_name, placement_group, job_id=None):
     """Call regionInstances.bulkInsert to create instances"""
-    assert len(nodes) > 0
-    if placement_group:
-        assert len(nodes) <= min(PLACEMENT_MAX_CNT, BULK_INSERT_LIMIT)
-    else:
-        assert len(nodes) <= BULK_INSERT_LIMIT
+    assert 0 < len(nodes) <= BULK_INSERT_LIMIT
```

Comment:
Testing the second assert later on will end up with some weird debug issues and I'm not sure what the model line will end up doing.

```
assert 0 < len(nodes) <= placement_group ? min(PLACEMENT_MAX_CNT, BULK_INSERT_LIMIT) : BULK_INSERT_LIMIT
```

Reply:
ack